src/Agent/NewRelic/Agent/Core/DataTransport/ServerlessModeDataTransportService.cs

```diff
@@ -161,8 +161,11 @@ public DataTransportResponseStatus Send(LogEventWireModelCollection loggingEvent
             return DataTransportResponseStatus.RequestSuccessful;
         }
 
-        // Not supported in serverless mode
-        public DataTransportResponseStatus Send(LoadedModuleWireModelCollection loadedModules, string transactionId) => throw new NotImplementedException();
+        public DataTransportResponseStatus Send(LoadedModuleWireModelCollection loadedModules, string transactionId)
+        {
+            // Not supported in serverless mode
+            return DataTransportResponseStatus.RequestSuccessful;
```

Comment:
Was this causing a problem? If we're generating module events in serverless mode something is probably wrong, so we may want to keep that as an exception.

Reply:
This was probably a leftover from my early testing where the agent wasn't really in serverless mode like I expected. I reverted the change.